### PR TITLE
Case 15127: Esc key doesn't consistently set my avatar to 'Away' mode

### PR DESCRIPTION
**Pull request by danteruiz**
Ticket - https://highfidelity.manuscript.com/f/cases/15127/Esc-key-doesn-t-consistently-set-my-avatar-to-Away-mode

### CODE REVIEW

File: interface/resources/qml/dialogs/CustomQueryDialog.qml [hunk 1]
```diff
@@ -273,11 +273,7 @@ ModalWindow {
             onTriggered: {
                 root.result = null;
                 root.canceled();
-                // FIXME we are leaking memory to avoid a crash
-                // root.destroy();
-
-                root.disableFade = true
-                visible = false;
+                root.destroy();
```
Comment (SamGondelman) [inline interface/resources/qml/dialogs/CustomQueryDialog.qml]: what about the crash?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17964/HighFidelity-Beta-PR15131-d1e376d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17964/HighFidelity-Quest-Beta-PR15131-d1e376d.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17964/HighFidelity-Beta-PR15131-d1e376d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17964/HighFidelity-Beta-Interface-PR15131-d1e376d.dmg

File: interface/resources/qml/dialogs/CustomQueryDialog.qml (hunk 1 above)
Comment (danteruiz) [reply interface/resources/qml/dialogs/CustomQueryDialog.qml]: no longer happens, this was fixed a while back and we didn't update these files

Comment (SamGondelman) [reply interface/resources/qml/dialogs/CustomQueryDialog.qml]: 👍

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17964/HighFidelity-Beta-PR15131-d1e376d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17964/HighFidelity-Beta-Interface-PR15131-d1e376d.exe

**Pull request merged**